src/local-time.lisp

```diff
@@ -235,9 +232,22 @@ describing a local time, or null to ask for the subzone after the last transitio
           (t
            (let* ((transitions (timezone-transitions timezone))
                   (unix-time (timestamp-values-to-unix seconds days))
-                  (transition-idx (transition-position unix-time transitions))
-                  (subzone-idx (elt indexes transition-idx))
-                  (subzone (elt subzones subzone-idx)))
+                  (transition-idx
+                   (transition-position (if guess-p
+                                            (- unix-time 86400)
```

Comment:
Use `+seconds-per-day+` here and below.

Reply:
I'm going to use +timezone-offset-min+ and +timezone-offset-max+ if you agree (a change that got lost somewhere).

Reply:
Yes, that makes more sense here. Perhaps those should be defined in terms of `+seconds-per-day+` too.

Reply:
Defining them in terms of `+seconds-per-day+` implies a relationship which does not exist; wouldn't that be misleading?

In the database we only find 'reasonable' high values around the 180° meridian. But what if someone would use offsets to model something like the change of calendar in Russia in 1917 (and why did nobody do it)?